APP_DEVELOPMENT.md

```diff
@@ -160,6 +174,31 @@ These are custom components we've developed, in addition to NativeBase readymade
 
 Displays the wide version of the STA logo
 
+#### STA Multi-coloured Matchstick Logo
+
+![STA Matchstick](app/src/Assets/Images/STA%20Matchstick%20Logo.png)
+
+STA Matchstick Logo with seven, rotated multi-coloured matchsticks
```

Comment:
I can see where you're coming from but this is actually a logo, not a component and doesn't belong here. I think other developers would be able to find this in the logos as that would probably the place where anyone would look for it.